src/Compute/Compute/Generated/Models/PSRestorePoint.cs

```diff
@@ -35,10 +35,7 @@ class PSRestorePoint
         //     for more details. Possible values include: 'CrashConsistent', 'FileSystemConsistent',
         //     'ApplicationConsistent'
         public string ConsistencyMode { get; set; }
-        //
-        // Summary:
-        //     Gets the provisioning details set by the server during Create restore point operation.
-        public RestorePointProvisioningDetails ProvisioningDetails { get; set; }
```

Comment:
Removing property in output type will cause breaking change. We cannot accept this PR unless major release.